app/routes/task_routes.py

```diff
@@ -1 +1,220 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response, jsonify
+from app.models.task import Task
+from ..db import db
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+import requests
+from app.routes.route_utilities import validate_model, check_complete
+
+load_dotenv()
+SLACKBOT_TOKEN = os.getenv("SLACKBOT_TOKEN")
+
+tasks_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+
+@tasks_bp.post("")
+def create_task():
+
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        return {"details": "Invalid data"}, 400
+    
+    if "description" not in request_body:
+        return {"details": "Invalid data"}, 400
+
+    if "completed_at" in request_body:
+        completed_at = request_body["completed_at"]
+
+    else:
+        completed_at = None
```

Comment:
We could condense all of this into a try/except block that would allow us to try and make the task. If any of the required keys were missing, it would throw an error! This would definitely cut down on these guard clauses.